corehq/apps/ota/views.py

```diff
@@ -328,3 +328,16 @@ def get_next_id(request, domain):
     if bucket_id is None:
         return HttpResponseBadRequest("You must provide a pool_id parameter")
     return HttpResponse(SerialIdBucket.get_next(domain, bucket_id, session_id))
+
+
+# Note: this endpoint does not require authentication
+@location_safe
+@require_GET
+def recovery_measures(request, domain, build_id):
+    app_id = get_app_cached(domain, build_id).master_id
+    response = {"app_id": request.GET.get('app_id')}
+    measures = [measure.to_mobile_json() for measure in
+                MobileRecoveryMeasure.objects.filter(domain=domain, app_id=app_id)]
```

Comment:
sorry for the late review but this seems like something that could easily be cached

Reply:
Good point - every user should see the same response for a given `(domain, build_id)`, so the whole response could be cached.  I'll submit a followup PR.